test-specifications/token-service/tokenMintTransaction.md

```diff
@@ -0,0 +1,200 @@
+| 8       | Mints an amount of 18,446,744,073,709,551,615 (uint64 max) fungible tokens     | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="18446744073709551615", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>] | The token mint fails with an INVALID_TOKEN_MINT_AMOUNT response code from the network.                          | N                 |
+| 9       | Mints an amount of 10,000 fungible tokens with 2 decimals                      | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="10000", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]                | The token mint succeeds and the token's treasury account contains 100 more minted tokens.                       | N                 |
+| 10      | Mints an amount of 10,000 fungible tokens with 1,000 max supply                | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="10000", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]                | The token mint fails with an INVALID_TOKEN_MINT_AMOUNT response code from the network.                          | N                 |
+| 11      | Mints fungible tokens with the treasury account frozen                         | tokenId=<CREATED_FUNGIBLE_TOKEN_ID>, amount="1000000", commonTransactionParams.signers=[<CREATED_FUNGIBLE_TOKEN_SUPPLY_KEY>]              | The token mint fails with an ACCOUNT_FROZEN_FOR_TOKEN response code from the network.                           | N                 |
+| 12      | Mints paused fungible tokens                                                   | tokenId=<PAUSED_FUNGIBLE_TOKEN_ID>, amount="1000000", commonTransactionParams.signers=[<PAUSED_FUNGIBLE_TOKEN_SUPPLY_KEY>]                | The token mint fails with an TOKEN_IS_PAUSED response code from the network.                                    | N                 |
+| 13      | Mints an amount of 1,000,000 NFTs                                              | tokenId=<CREATED_NON_FUNGIBLE_TOKEN_ID>, amount="1000000", commonTransactionParams.signers=[<CREATED_NON_FUNGIBLE_TOKEN_SUPPLY_KEY>]      | The token mint fails with an INVALID_TOKEN_MINT_METADATA response code from the network.                        | N                 |
+
+#### JSON Request Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 64362,
+  "method": "mintToken",
+  "params": {
+    "tokenId": "0.0.15432",
+    "amount": "1000000",
+    "commonTransactionParams": {
+      "signers": [
+        "302E020100300506032B657004220420DE6788D0A09F20DED806F446C02FB929D8CD8D17022374AFB3739A1D50BA72C8"
+      ]
+    }
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 64362,
+  "result": {
+    "status": "SUCCESS",
+    "newTotalSupply": "2000000"
+  }
+}
+```
+
+
+### **Metadata:**
+
+- The metadata for each NFT to be minted.
```

Comment:
Do we test the limits of metadata content at all? For example, do we test empty string, a string with many characters, or a string with several multibyte characters (e.g. `𝐇𝐞𝐝𝐞𝐫𝐚 𝐡𝐚𝐬𝐡𝐠𝐫𝐚𝐩𝐡 𝐭𝐞𝐬𝐭 𝐝𝐚𝐭𝐚`, none of which are ASCII letters)?

Reply:
Added more test cases to test these points